src/components/Moon.js

```diff
@@ -1,12 +1,21 @@
-import React from 'react'
-import moon from '../assets/moon.png'
-import { nanoid } from 'nanoid'
+import React, { useEffect } from 'react'
 
+import moon from '../assets/images/moon.png'
+import { useState } from 'react'
+import { useLocation } from 'react-router-dom'
 
-export const Moon = () => { 
-  const id = nanoid(10)
-  return (
-    <img alt="moon" src={moon} key={id} className="moon"/>
-  )
-}
+export const Moon = ({ id }) => {
+	const [isInactive, setInactive] = useState(false)
+	const location = useLocation().pathname.split('/')[2]
+
+	useEffect(
```

Comment:
I haven't investigated this entirely, but my guess is that it could be done in a different way.
Not that any one way is better, but I think I've done something like this by mutating state:
```
{
  active: false,
}

state.active = !state.active
```

Reply:
Would this be something I would use with or without useState?

Reply:
with

Reply:
I feel like I have attempted maniupulating useState like that in the past but I don't like having unused variables which is why I always use the setWhatever. How do you go around that?